### PR TITLE
feat: [CVS-9009]: Updated CV icon

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:

Screenshot 2022-01-31 at 11 01 46 PM

Screenshot 2022-01-31 at 9 10 37 PM
Screenshot 2022-01-31 at 9 08 30 PM

Screenshot 2022-01-31 at 9 08 19 PM
Screenshot 2022-01-31 at 9 08 13 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by anjanBalgovind — DISMISSED** [review]

**Comment by deepesh-ui** [comment]
trigger cypress

File: src/modules/10-common/pages/welcome/WelcomePage.module.scss.d.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ declare const styles: {
   readonly changeSelection: string
   readonly container: string
   readonly continueButton: string
+  readonly cvIconText: string
```
Comment (praneshg239) [inline src/modules/10-common/pages/welcome/WelcomePage.module.scss.d.ts:16]: why `CV` here?

Comment (deepesh-ui) [reply src/modules/10-common/pages/welcome/WelcomePage.module.scss.d.ts:16]: all places variables are still prefixed as cv so keeping same

**Review by praneshg239 — APPROVED** [review]

File: src/modules/45-projects-orgs/utils/utils.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const getModulePurpose = (module: ModuleName): keyof StringsMap | undefin
     case ModuleName.CD:
       return 'common.purpose.cd.delivery'
     case ModuleName.CV:
-      return 'common.purpose.cv.intelligence'
+      return 'common.purpose.ce.management'
```
Comment (harshilbits) [inline src/modules/45-projects-orgs/utils/utils.ts:45]: Just curious to know why we are keeping under ce ?

Comment (deepesh-ui) [reply src/modules/45-projects-orgs/utils/utils.ts:45]: Management is common string for both **Cloud Cost Management** and **Service Reliability Management**

**Review by harshilbits — APPROVED** [review]

**Pull request merged**